README.md

```diff
@@ -2,7 +2,19 @@
 
 # Fairness in machine learning - Mitigation algorithms
 
-A Python package that implements a variety of algorithms that mitigate unfairness in supervised machine learning including:
+A Python package that implements a variety of algorithms that mitigate unfairness in supervised machine learning.
+
+## Current Release
+
+*  The current stable release is available at [FairLearn v0.2.0](https://github.com/microsoft/fairlearn/tree/v0.2.0 ).
+
+* This repo contains our latest development work and is frequently updated.
+
+* Current users can reference [Existing users: How to onboard to fairlearn v0.3+](#existing).
+
```

Comment:
Change this bullet to something like "Our latest work differs substantively from version 0.2.0, please see [Existing...]"

Reply:
done